concepts/Common/Security Considerations/30 Export Vulnerabilities/05 CSV Injection.md

```diff
@@ -0,0 +1,191 @@
+If you export data from DataGrid or PivotGrid in CSV format, you should take a thread of a CSV Injection Attack (also known as a formula injection attack). Such attacks involve an injection of a malicious character sequence that is interpreted as a formula and executed within a computer network. Cell values that start with the **=**, **+**, **-**, and **@** characters can initiate an injection attack.
```

Comment:
"If you export data from DataGrid or PivotGrid in CSV format, you should take a thread of a CSV Injection Attack (also known as a formula injection attack)."

What do you mean by "a thread" in the context of a CSV Injection Attack?

Reply:
Hello Albert,

My bad, looks like I haven't finished editing this line. Actually, I meant the following instead:

```suggestion
If you export data from DataGrid or PivotGrid in CSV format, you should take a threat of a CSV Injection Attack (also known as a formula injection attack) into account. Such attacks involve an injection of a malicious character sequence that is interpreted as a formula and executed within a computer network. Cell values that start with the **=**, **+**, **-**, and **@** characters can initiate an injection attack.
```

Reply:
Thank you very much! I should have done so in the first place...